data/blog/2024-year-in-review.mdx

```diff
@@ -0,0 +1,545 @@
+---
+title: "OpenSats' 2024 Year in Review"
+date: '2024-12-30'
+tags: ['OpenSats', 'grants', 'donations', 'bitcoin', 'nostr', 'review']
+draft: false
+authors: ['dergigi', 'default']
+images: ['/static/images/blog/66-opensats-2024-year-in-review.jpg']
+summary: "365 days. Two funds. 200+ grants. Billions of sats."
+---
+
+Twenty-twenty-four. What a year! A lot has happened in the past 12 months, and
+just like last year, we want to take the opportunity to pause, reflect, and
+share our year in review. We want to look back at the announcements we made,
+share an update in regards to the donations we received, and give an overview of
+how we allocated our funds. After that we will share updates to our grantee
+statistics and provide an outlook of what the next year might bring.
+
+- [Announcements](#announcements)
+- [Donations](#donations)
+- [Allocation](#allocation)
+- [Grantees](#grantees)
+- [Outlook](#outlook)
+
+But before we dive into the various graphs and numbers, let us spend a minute to
```

Comment:
```suggestion
Before we dive into the various graphs and numbers, let us spend a minute to
```

Reply:
I think I'll keep the but(t) 😜

Reply:
Agree that best not to begin a sentence with "But"; for instance, I think this would be more correct: s/But/However, /